docs/www/data-transform.md

```diff
@@ -0,0 +1,387 @@
+---
+title: Data transformation
+order: 0
+---
+# Data transformation
+
+**Important** - This feature is in technical preview; a cluster with this feature enabled is not yet supported for production.
+
+Redpanda Data Transforms enable users to create WebAssembly-based scripts to perform simple data transformations on topics. 
+
+This eliminates "data ping-pong" by removing the need to consume data streams from a separate system for simple transformations such as scrubbing or cleaning data.
+
+With Redpanda, you can now deploy custom Node.js programs to handle the entire consume-transform-produce cycle for you.
+
+We do all of this co-processing using a new transformation engine built on-top of [WASM (WebAssembly)](https://developer.mozilla.org/en-US/docs/WebAssembly) technology.
+
+All you need to do is to provide:
+- A JavaScript code that transforms the data. 
+- The source topics where to consume from.
+  
+The destination topics can be generated on demand when your transformation runs. 
+
+You can jump start your data transformation development with `rpk`. We provide a sample Node.js project so you can start developing right away. Then you can deploy the data transform to Redpanda for all of the brokers to run.
+
+Be aware that even though we're referring to this feature as data transformation, the Kafka topics are still immutable. 
+What happens under the hood, is that we're reading from an immutable Kafka topic, applying a data transformation process and then we producing to another immutable Kafka topic.
+Your data is never actually transformed, but rather read and written again. 
+
+## Enable data transforms
+
+To enable the coprocessor that runs the data transformation you need to enable the engine in the Redpanda configuration file. 
+
+In your `redpanda.yaml` file (which the default path is `/etc/redpanda/redpanda.yaml`) add these **required** flags under `redpanda`.
```

Comment:
```suggestion
In your `redpanda.yaml` file (which the default path is `/etc/redpanda/redpanda.yaml`) you must first set the `auto_create_topics_enabled` parameter to `true`. The default for this parameter is `false`. The default is set to `false` to prevent a user from accidentally creating the output topic as a normal topic because the coprocessor attempts to create it from scratch. If `auto_create_topics_enabled` is set to `false`, the coproc will return an error that you are attempting to push data to a normal topic that already exists. 


In addition, you must add these **required** flags under `redpanda` in the `redpanda.yaml file`.
```

@graphcareful @VadimPlh - I pretty much just copied what you said. But does this text work?